src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public int getRandomBall() {
+        Random randomBall = new Random();
+        int value = randomBall.nextInt(100);
+        return value;
+    }
+}
```

Comment:
The `getRandomBall` method is creating a new instance of `Random` every time it is called. This is not necessary and can be inefficient. Consider creating a single `Random` instance as a class-level field and reusing it.